storm-core/test/jvm/org/apache/storm/scheduler/resource/TestUtilsForResourceAwareScheduler.java

```diff
@@ -109,7 +112,7 @@
     public static Map<ExecutorDetails, String> genExecsAndComps(StormTopology topology, int spoutParallelism, int boltParallelism) {
         Map<ExecutorDetails, String> retMap = new HashMap<ExecutorDetails, String>();
         int startTask = 0;
-        int endTask = 1;
+        int endTask = 0;
```

Comment:
This corrects the executor number in tests to be [0,0] [1,1]; which used to be [0, 1] [1,2], not actually what we have in Storm.